src/wh_server_nvm.c

```diff
@@ -0,0 +1,311 @@
+
+/* System libraries */
+#include <stdint.h>
+#include <stdlib.h>  /* For NULL */
+#include <string.h>  /* For memset, memcpy */
+
+/* Common WolfHSM types and defines shared with the server */
+#include "wolfhsm/wh_error.h"
+#include "wolfhsm/wh_comm.h"
+
+#include "wolfhsm/wh_nvm.h"
+
+#include "wolfhsm/wh_message.h"
+#include "wolfhsm/wh_message_nvm.h"
+#include "wolfhsm/wh_server.h"
+
+int wh_Server_HandleNvmRequest(whServerContext* server,
+        uint16_t magic, uint16_t action, uint16_t seq,
+        uint16_t req_size, const void* req_packet,
+        uint16_t *out_resp_size, void* resp_packet)
+{
+    int rc = 0;
+    switch (action) {
+
+    case WH_MESSAGE_NVM_ACTION_INIT:
```

Comment:
Originally, I had expected the NVM Init client call to provide some identity/authentication so that the server understands who is on the other side of the connection and could use that to help mask which objects it has access to.  Cleanup was expected to inform the server that no further authenticated access is necessary.  Just a placeholder for now.